white_label/lib/white_label/stubs.rb

```diff
@@ -238,6 +245,7 @@ def self.default(visited=[])
       end
 
       def self.stub(http_resp, stub:)
+        data = {}
```

Comment:
Is this a bug fix? How are these being added?

Reply:
This was actually added in the documentation update that I did. 
This code was incorrect (data did not exist and so these would error out).  I made a few changes in the Base builders that made this a bit more consistent so that implementations that extend them don't need to initialize data themselves.